### PR TITLE
Misc doc updates

### PR DESCRIPTION
**Pull request by ankith26**
Some more minor misc doc updates. I find myself at pygame docs most of the time, so I am polishing them a bit to help people like me :)

### CODE REVIEW

**Comment by ankith26** [comment]
Just moved the version added tag to the bottom of their respectively blocks, to put it inline with docs for functions. Added video event and mousewheel to event docs and misc fixes

**Comment by ankith26** [comment]
Just improving docs, mainly for the display, mouse and event modules. Many nitpicky fixes and also many useful and important additions

**Comment by ankith26** [comment]
Latest commit closes #1166

**Review by MyreMylar — APPROVED** [review]
All seems great to me! I built the docs locally on windows and checked against the docs online to confirm the changes.

Thanks for the contribution and tidying up these docs. Approved 👍

**Comment by ankith26** [comment]
Thanks for reviewing :)

File: docs/reST/ref/display.rst [hunk 1]
```diff
@@ -42,6 +42,42 @@ user adjusts the window dimensions. Hardware displays that draw direct to the
 screen will get ``pygame.VIDEOEXPOSE`` events when portions of the window must
 be redrawn.
 
+In pygame 2 (when built with SDL 2), there is a new type of event called
```
Comment (illume) [inline docs/reST/ref/display.rst:45]: For the whole of the pygame documentation (and not just here) we should decide on how we talk about new things and changes. It's easiest IMHO to just talk about pygame versions, and not add anything about SDL 2. 
- Adding both is redundant most of the time.
- Probably most readers won't know what SDL 2 is, and it isn't relevant to them.
- We will not continue to support SDL1 builds of pygame, it's only useful for development.
- Keeping it to 'what pygame version do I have?' makes it simpler for people.

File: docs/reST/ref/display.rst [hunk 2]
```diff
@@ -42,6 +42,42 @@ user adjusts the window dimensions. Hardware displays that draw direct to the
...
+While SDL has removed support for these old events, pygame is still supporting
+them for backwards compatability.
+The new events of type ``pygame.WINDOWEVENT`` have an ``event`` attribute that 
+can take the following values.
+
+::
+
+   Value of event attribute      Short description
+         
+   WINDOWEVENT_SHOWN            Window became shown
```
Comment (illume) [inline docs/reST/ref/display.rst:58]: The docs are good. 

So good, I think we need to look again at WINDOWEVENT. I don't think it recieved enough attention when we added it.

Doing a repr(event) on it, shows stuff like <WindowEvent 'event': 3>, which is meaningless.

Also it's quite verbose to use:
` event.type == pygame.WINDOWEVENT and event.event == pygame.WINDOWEVENT_HIDDEN:`
VS using attributes:
`if event.type == pygame.WINDOWEVENT and event.hidden:`
VS using a separate event type for each one:
`if event.type == pygame.WINDOWEVENT_HIDDEN:`

Having to handle the WINDOWEVENT differently than all the other types is an extra load on the user.

File: docs/reST/ref/mouse.rst [hunk 3]
```diff
@@ -58,13 +60,14 @@ to access data about the mouse scroll, such as ``which`` (it will tell you what
...
-               if event_var.type == QUIT:
+         for event in pygame.event.get():
+               if event.type == QUIT:
                   pygame.quit()
                   return
-               elif event_var.type == MOUSEWHEEL:
-                  print(event_var) # can access properties with prop notation 
-                                   # (ex: event_var.y)
+               elif event.type == MOUSEWHEEL:
+                  print(event)
```
Comment (illume) [inline docs/reST/ref/mouse.rst:68]: Perhaps printing the mouse wheel attributes would be more instructive for the reader.
`print(event.x, event.y, event.flipped, event.which)`

When I went to look up the event MOUSEWHEEL event on the event page of the documentation I didn't see the attributes listed.

From event.c I see these attributes (flipped, x, y, which).
```c
        case SDL_MOUSEWHEEL:
            /* https://wiki.libsdl.org/SDL_MouseWheelEvent */
#ifndef NO_SDL_MOUSEWHEEL_FLIPPED
            _pg_insobj(dict, "flipped", PyBool_FromLong(event->wheel.direction == SDL_MOUSEWHEEL_FLIPPED));
#else
            _pg_insobj(dict, "flipped", PyBool_FromLong(0));
#endif
            _pg_insobj(dict, "y", PyInt_FromLong(event->wheel.y));
            _pg_insobj(dict, "x", PyInt_FromLong(event->wheel.x));
            _pg_insobj(dict, "which", PyInt_FromLong(event->wheel.which));
            break;
```

File: docs/reST/ref/mouse.rst [hunk 4]
```diff
@@ -73,31 +76,29 @@ to access data about the mouse scroll, such as ``which`` (it will tell you what
 .. function:: get_pressed
 
    | :sl:`get the state of the mouse buttons`
-   | :sg:`get_pressed(num_buttons=3) -> (button1, button2, button3)
+   | :sg:`get_pressed(num_buttons=3) -> (button1, button2, button3)`
```
Comment (illume) [inline docs/reST/ref/mouse.rst:79]: 👍

File: docs/reST/ref/event.rst [hunk 5]
```diff
@@ -86,13 +86,13 @@ specific attributes.
     VIDEOEXPOSE       none
     USEREVENT         code
 
+.. versionchanged:: The ``joy`` attribute was deprecated, ``instance_id`` was added.
```
Comment (illume) [inline docs/reST/ref/event.rst:89]: This is missing a version number.

File: docs/reST/ref/display.rst (hunk 1 above)
Comment (ankith26) [reply docs/reST/ref/display.rst:45]: I guess that throughout the docs we can replace sdl2 with pygame 2, just to make it easier for users. I will open a PR for this

File: docs/reST/ref/display.rst (hunk 2 above)
Comment (ankith26) [reply docs/reST/ref/display.rst:58]: Hmm, that would involve a lot of code change though

File: docs/reST/ref/mouse.rst [hunk 6]
```diff
@@ -58,13 +60,14 @@ to access data about the mouse scroll, such as ``which`` (it will tell you what
    
    def main():
       while True:
-         for event_var in pygame.event.get():
-               if event_var.type == QUIT:
+         for event in pygame.event.get():
```
[the remark this answers is not in the document]
Comment (ankith26) [reply docs/reST/ref/mouse.rst:63]: :)

File: docs/reST/ref/mouse.rst (hunk 3 above)
Comment (ankith26) [reply docs/reST/ref/mouse.rst:68]: Done :)

File: docs/reST/ref/mouse.rst (hunk 4 above)
Comment (ankith26) [reply docs/reST/ref/mouse.rst:82]: :)

File: docs/reST/ref/event.rst (hunk 5 above)
Comment (ankith26) [reply docs/reST/ref/event.rst]: Done. 👍

File: docs/reST/ref/display.rst (hunk 2 above)
Comment (illume) [reply docs/reST/ref/display.rst:58]: Added an issue about this here: https://github.com/pygame/pygame/issues/2214

File: docs/reST/ref/display.rst (hunk 1 above)
Comment (illume) [reply docs/reST/ref/display.rst:45]: There's a separate PR ongoing for this here: https://github.com/pygame/pygame/pull/2211

**Review by illume — APPROVED** [review]
Thanks :)

**Pull request merged**